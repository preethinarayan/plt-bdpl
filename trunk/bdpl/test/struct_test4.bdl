struct Outer
{
	int b_1;
	struct Inner
	{
		byte i_2_1;
		byte i_2_2;
	} inner_ob_2;
	int c_3;
} ob1;

struct Outer1
{
	int b_1;
	struct Outer
	{
		byte i_2_1;
		byte i_2_2;
	} inner_ob_2;
	int c_3;
	struct Inner
	{
		int huh;
	}na;
	type struct Outer n_outer;
	type struct Inner new_inner;
} ob2;


type struct Outer1 new_ob;
type struct Outer new_ob1;
