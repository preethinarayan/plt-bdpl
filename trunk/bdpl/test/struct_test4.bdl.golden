struct:Outer
{
	b_1 => 0,
	inner_ob_2 => struct:Inner
	{
		i_2_1 => 0x0,
		i_2_2 => 0x0,
	}
	,
	c_3 => 0,
}
